skbio/stats/distance/_permdisp.py

```diff
@@ -170,7 +170,7 @@ def permdisp(distance_matrix, grouping, column=None, test='median',
     statistics.
 
     >>> np.random.seed(0)
-    >>> permdisp(dm, grouping, test='centroid', permutations=6)
+    >>> permdisp(dm,grouping,test='centroid',permutations=6) #doctest:+ELLIPSIS
```

Comment:
I don't think there's need to remove the spaces between arguments, right?
```suggestion
    >>> permdisp(dm, grouping, test='centroid', permutations=6) #doctest:+ELLIPSIS
```